src/middlewares/isBlocked.js

```diff
@@ -0,0 +1,63 @@
+import models from '../api/models';
+import status from '../helpers/constants/status.codes';
+
+const { User } = models;
+/**
+ * check if user is blocked
+ *
+ * @export
+ * @class checkIfBlocked
+ */
+class checkIfBlocked {
+  /**
+   * this is a middleware which checks if a user is blocked.
```

Comment:
same here @Ramadhan0

Reply:
This middleware will check if a user is already blocked because we don't have to block a user who is already blocked